testsuite/model/src/test/java/org/keycloak/testsuite/model/session/OfflineSessionPersistenceTest.java

```diff
@@ -130,6 +131,7 @@ public void testPersistenceSingleNode() {
     }
 
     @Test(timeout = 90 * 1000)
+    @Ignore("test currently fails as offline sessions are not loaded by default any more?!")
```

Comment:
@martin-kanis - I've been updating inIndependentFactories() so that any assert wrapped inside it will make the test fail. Due to this change, the following code now fails when "keycloak.model.parameters=Infinispan,Jpa" are set.

I now wonder: is this due to offline sessions no longer being loaded by default, one of the PR's that went through your hands lately?

```
inIndependentFactories(4, 30, () -> assertOfflineSessionsExist(realmId, clientSessionIds.keySet()));
```

Reply:
After quick debugging it seems no offline user sessions are created. I suspect there's a bug in this block https://github.com/keycloak/keycloak/pull/10288/files#diff-ab99606564bc3505e0948722bd230a49e89252a990a279c078445d32c4d259c8R150-R161
When I evaluate `session.sessions().getOfflineUserSessionsStream(realm, session.users().getUserById(realm, userIds.get(0))).collect(Collectors.toList())` at the end of the block, there are no sessions.

Reply:
I looked at the test again today, and found that the stream returned by createOfflineSessions was never consumed, therefore no offline sessions were created. Will push a change once the current run completes (don't want to interrupt it).